styles/sass/partials/_drinkList.scss

```diff
@@ -0,0 +1,26 @@
+.drink__list {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: 50px;
+  &--result {
+    flex: 0 1 calc(100%/5 - 50px);
+    display: flex;
```

Comment:
I don't believe these display, justify, and align are necessary, as you're using absolute positioning on the name.